server/server.go

```diff
@@ -51,7 +51,8 @@ type Options struct {
 	DefaultSamplingPolicy trace.Sampler
 }
 
-// New creates a new server. New(nil) is the same as new(Server).
+// New creates a new server. New(nil) is the same as new(Server). Note: A
+// configured Requestlogger will not log HealthChecks.
```

Comment:
This seems like a more appropriate message under `ListenAndServe`. WDYT?

Reply:
Both seem fine to me. I'll update.